### PR TITLE
Update benchmark and control structure for NSA CISA Kubernetes Hardening v1

### PR DESCRIPTION
**Pull request by cbruno10**

### CODE REVIEW

File: mod.sp [hunk 1]
```diff
@@ -5,11 +5,11 @@ mod "kubernetes_compliance" {
   color         = "#0089D6"
   documentation = file("./docs/index.md")
   icon          = "/images/mods/turbot/kubernetes-compliance.svg"
-  categories    = ["kubernetes", "compliance", "public cloud", "security"]
+  categories    = ["kubernetes", "compliance", "software development", "security"]
 
   opengraph {
     title       = "Steampipe Mod for Kubernetes Compliance"
-    description = "Run individual configuration, compliance and security controls across all your Kubernetes accounts using Steampipe."
+    description = "The National Security Agency (NSA) and CISA Kubernetes Hardening Guidance, describes the security challenges associated with setting up and securing a Kubernetes cluster, and presents hardening strategies to guide system administrators in avoiding common misconfigurations."
```
Comment (cbruno10) [inline mod.sp:12]: This description, along with the mod's description above, should match the repo's description: `Run individual controls or full compliance benchmarks for NSA CISA Kubernetes Hardening Guidance across all of your Kubernetes clusters using Steampipe.`

File: nsa_cisa_v1/docs/nsa_cisa_kubernetes_hardening_overview.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-To get the latest version of the offical guide, please visit [here](https://media.defense.gov/2021/Aug/03/2002820425/-1/-1/1/CTR_KUBERNETES%20HARDENING%20GUIDANCE.PDF)
+To get the latest version of the offical guide, please visit [here](https://media.defense.gov/2021/Aug/03/2002820425/-1/-1/1/CTR_KUBERNETES%20HARDENING%20GUIDANCE.PDF).
```
Comment (cbruno10) [inline nsa_cisa_v1/docs/nsa_cisa_kubernetes_hardening_overview.md:1]:
```suggestion
To get the latest version of the official guide, please visit [here](https://media.defense.gov/2021/Aug/03/2002820425/-1/-1/1/CTR_KUBERNETES%20HARDENING%20GUIDANCE.PDF).
```

File: nsa_cisa_v1/nsa_cisa_v1.sp [hunk 3]
```diff
@@ -0,0 +1,16 @@
+locals {
+  nsa_cisa_v1_common_tags = {
+    nsa_cisa_v1 = "true"
+    plugin      = "kubernetes"
+  }
+}
+
+benchmark "nsa_cisa_v1" {
+  title         = "NSA and CISA Kubernetes Hardening Guidance v1.0"
+  documentation = file("./nsa_cisa_v1/docs/nsa_cisa_kubernetes_hardening_overview.md")
```
Comment (cbruno10) [inline nsa_cisa_v1/nsa_cisa_v1.sp:10]:
```suggestion
  documentation = file("./nsa_cisa_v1/docs/nsa_cisa_v1_overview.md")
```

The file name should be updated too

File: nsa_cisa_v1/nsa_cisa_v1_pod_security.sp [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+    benchmark.nsa_cisa_v1_pod_security_container_security_service_hardening,
+    benchmark.nsa_cisa_v1_pod_security_host_network_access_disabled,
+    benchmark.nsa_cisa_v1_pod_security_hostpid_hostipc_sharing_disabled,
+    benchmark.nsa_cisa_v1_pod_security_immutable_container_filesystem,
+    benchmark.nsa_cisa_v1_pod_security_non_root_container,
+    benchmark.nsa_cisa_v1_pod_security_service_account_token_disabled,
+  ]
+}
+
+// Container disallow host path benchmark
```
Comment (cbruno10) [inline nsa_cisa_v1/nsa_cisa_v1_pod_security.sp:17]: Now that we've moved the controls out, do we need these comments in anymore? I'm ok to leave them in if helpful, but I know they were more helpful when the file was much larger.

**Review by cbruno10 — COMMENTED** [review]
@bigdatasourav Please take a look at the comments, thanks!

File: nsa_cisa_v1/nsa_cisa_v1_pod_security.sp (hunk 4 above)
Comment (bigdatasourav) [reply nsa_cisa_v1/nsa_cisa_v1_pod_security.sp:17]: I think we can remove those.

File: control/local.sp [hunk 5]
```diff
@@ -0,0 +1,25 @@
+locals {
+  title_container_cpu_limit                     = "__KIND__ containers should have a CPU limit"
```
Comment (cbruno10) [inline control/local.sp:2]:
```suggestion
  container_cpu_limit_title                           = "__KIND__ containers should have a CPU limit"
  container_cpu_limit_desc                          = "__KIND__ containers should have a CPU limit"

  container_cpu_request_title                     = "__KIND__ containers should have a CPU limit"
  container_cpu_request_desc                   = "__KIND__ containers should have a CPU limit"

  ...
```

File: control/replicaset.sp [hunk 6]
```diff
@@ -0,0 +1,69 @@
+control "replicaset_cpu_limit" {
+  title       = replace(local.title_container_cpu_limit, "__KIND__", "ReplicaSet")
```
Comment (cbruno10) [inline control/replicaset.sp:2]:
```suggestion
  title       = replace(local.container_cpu_limit_title, "__KIND__", "ReplicaSet")
```

**Pull request merged**